pysmt/solvers/msat.py

```diff
@@ -362,9 +374,7 @@ def set_preferred_var(self, var, val=None):
     def print_model(self, name_filter=None):
         if name_filter is not None:
             raise NotImplementedError
-        for v in self.converter.symbol_to_decl.keys():
```

Comment:
Why this change?

Reply:
I noted that there was a misuse of `symbol_to_decl`: sometimes it was used as a map from symbol names (strings) to decls and sometimes as a map from symbols (FNode) to decls. This uniforms the use avoiding some cache miss.